packages/webpack-cli/lib/groups/OutputGroup.js

```diff
@@ -27,7 +27,10 @@ class OutputGroup extends GroupHelper {
         const { args } = this;
         if (args) {
             const { output } = args;
-            if (!output) {
+            // TODO: Remove comment before merge
+            // We need to show warning when empty output flag is supplied
+            // which is set to boolean true by commander
```

Comment:
/cc @Loonride should we remove it before merge?

Reply:
Yes, removed. I couldn't seem to get it to be set to true as @rishabh3112 wrote. Here is the test I added:

https://github.com/webpack/webpack-cli/pull/1481/files#diff-2cb16055e37ba4584d42ff3630696acdR78-R82

Reply:
@Loonride The check can be removed now.

P.S.
This was because I was kept value for flag optional when I implemented `arg-parser` initially.
It was this (in `arg-parser.js` where we are registering options),
```
 const flagsWithType = option.type !== Boolean ? flags + ' [type]' : flags;
```
Now it is using required argument `<value>`.